### DISCUSSION TITLE
UserCSS Editor enabled by default

### DISCUSSION THREAD

**Discussion by pabli24** [openstyles/stylus#1714] [Ideas]
Why the usercss editor is not the default one yet?

Userstyles org styles are converted to usercss anyways.
Having two editors can be very confusing for new users.
Many people are forgetting to enable it, and try to write usercss in the wrong editor.

**Comment by tophf** [comment]
New users aren't supposed to have read usercss documentation and know what `@-moz-document` is.

**Reply by tophf** [reply]
I guess I'll add detection of metadata comments inside sections and display a warning if found.

**Reply by tophf** [reply]
Done: 9457a21d
